doc/Building_FRR_on_Ubuntu1604.md

```diff
@@ -0,0 +1,111 @@
+Building FRR on Ubuntu 12.04LTS from Git Source
+===============================================
+
+- MPLS is not supported on `Ubuntu 12.04` with default kernel. MPLS requires 
```

Comment:
I found some more info about it:
http://www.cnx-software.com/2017/02/13/how-to-upgrade-to-linux-4-8-in-ubuntu-16-04-2/